cli/batch/cli.py

```diff
@@ -0,0 +1,52 @@
+"""
+Provide implementation of the command line interface's batch commands.
+"""
+import asyncio
+import re
+import sys
+
+import click
+from remme import Remme
+
+from cli.batch.help import BATCH_ID_ARGUMENT_HELP_MESSAGE
+from cli.batch.service import Batch
+from cli.constants import (
+    BATCH_ID_REGEXP,
+    FAILED_EXIT_FROM_COMMAND,
+    NODE_URL_ARGUMENT_HELP_MESSAGE,
+)
+from cli.utils import dict_to_pretty_json
+
+loop = asyncio.get_event_loop()
+
+
+@click.group('batch', chain=True)
+def batch_commands():
+    """
+    Provide commands for working with batches.
+    """
+    pass
+
+
+@click.option('--id', type=str, required=True, help=BATCH_ID_ARGUMENT_HELP_MESSAGE)
+@click.option('--node-url', type=str, required=False, help=NODE_URL_ARGUMENT_HELP_MESSAGE)
+@batch_commands.command('get-single')
+def get_batch(id, node_url):
+    """
+    Get batch by identifier.
```

Comment:
Or move to `Get batch by its identifier` or another example (in the docs) use `Get batch by identifier`. I just want to unify anywhere.

Reply:
Done.